config/routes.rb

```diff
@@ -171,6 +171,7 @@
       get  "/payment_preferences"                     => "payment_preferences#index"
       put  "/payment_preferences"                     => "payment_preferences#update"
       # PayPal Connect
+      get  "/paypal_preferences" => redirect("/%{locale}/admin/payment_preferences")
```

Comment:
Will this work now even without the Stripe feature flag? From now on, the payment preferences are always in this URL with or without stripe flag enabled?